manifests/init.pp

```diff
@@ -13,13 +13,13 @@
 # @param gpgkeyfingerprint String with the ID from the gpg key that signed the packages
 # @param ensure Install or deinstall all related components
 class lldpd (
-  Boolean $manage_facts,
-  Boolean $manage_service,
-  Boolean $manage_repo,
-  Boolean $manage_jq,
-  Optional[String] $repourl,
-  String $gpgkeyfingerprint,
-  Enum['present', 'absent', 'latest'] $ensure,
+  Boolean $manage_facts = true,
+  Boolean $manage_service = true,
+  Boolean $manage_repo = true,
```

Comment:
Since many override this, I'd actually not have a default here at all. Explicitly set it for every distro.

Reply:
sounds good, will do. I idea was to have it here to remove duplicate hiera entries.